run-development.sh

```diff
@@ -64,7 +66,16 @@ fi
 # Start server
 ##############
 
-cd /usr/src/build/
-/usr/src/app/node_modules/.bin/babel-node \
-    --inspect="0.0.0.0:9229" \
-    ./app.js
+cd /usr/src/dist/
+if [ "$NO_BABEL_NODE" == "true" ]
```

Comment:
less savvy ides have a problem with debugging in babel node. I don't understand why we are running in babel-node again as we have already babelified before, but this is the current behaviour so i'm leaving it unchanged.

added an env_var to tell the container to just use regular node so less savvy ides are also happy